eng/scripts/Generate-Dependency-Test-References.ps1

```diff
@@ -0,0 +1,75 @@
+<#
```

Comment:
It would be good to write some unit tests for this script. I have some WIP guidance on this [here](https://github.com/Azure/azure-sdk-tools/blob/75f0ce338829ea4b6af65f4850022494811fdf80/doc/development/powershell.md#unit-testing).

Reply:
Working on the test.